main.go

```diff
@@ -56,6 +56,135 @@ var (
 		"Task memory RSS usage in bytes.",
 		variableLabels, nil,
 	)
+
+	frameworkLabels = []string{"id", "name"}
+
+	frameworkResourcesUsedCPUs = prometheus.NewDesc(
+		"mesos_framework_resources_used_cpus",
+		"Fractional CPUs used by a framework.",
+		frameworkLabels, nil,
+	)
+
+	frameworkResourcesUsedDisk = prometheus.NewDesc(
+		"mesos_framework_resources_used_disk_megabytes",
```

Comment:
Let's keep this in base units (`_bytes`). Easier to correlate with disk size metrics from other exporters which are all in bytes, plus otherwise it's not 100% clear whether it's MiB or MB otherwise. Probably MiB here, so multiply by 1024^2?

Same for the one below.